File: tests/unit/backend/jasmine-node/simple/simple-spec.js
var fs = require('../../../../../src/backend/nodejs/simple');

describe('Simple spec', function() {
  it('jasmine-node works', function() {
    expect(1).toBe(1);
  });

  it('sum wors', function() {
    var result = fs.sum(1, 2);
    expect(result).toBe(3);
  });
});

File: tests/unit/backend/plain-ruby/rspec/simple/simple_spec.rb
root = '../../../../../../'

require_relative(root + 'src/backend/plain-ruby/simple/simple')

describe 'sum method' do
  it 'returns the right values' do
    expect(sum(1,2)).to eq(3)
    expect(sum(0,0)).to eq(0)
    expect(sum(-1,-3)).to eq(-4)
  end
end